README.md

```diff
@@ -136,101 +136,7 @@ available parameters are in [the reference documentation][pkg-config].
 
 Interesting things can be done with `footloose`!
 
-## Customize the OS image
-
-It is possible to create docker images that specialize a [`footloose` base
-image](https://github.com/weaveworks/footloose#choosing-the-os-image-to-run) to
-suit your needs.
-
-For instance, if we want the created machines to run `fedora29` with the
-`htop` package already pre-installed:
-
-```Dockerfile
-FROM quay.io/footloose/fedora29
-
-# Pre-seed the htop package
-RUN dnf -y install htop && dnf clean all
-
-```
-
-Build that image:
-
-```console
-docker build -t fedora29-htop .
-```
-
-Configure `footloose.yaml` to use that image by either editing the file or running:
-
-```console
-footloose config create --image fedora29-htop
-````
-
-`htop` will be available on the newly created machines!
-
-```console
-$ footloose create
-$ footloose ssh root@node0
-# htop
-```
-
-## Run Apache with Footloose
-
-Using the footloose base like above, create a docker file which installs Apache and
-exposes a port like 80 or 443:
-
-```Dockerfile
-FROM quay.io/footloose/ubuntu18.04
-
-RUN apt-get update && apt-get install -y apache2
-COPY index.html /var/www/html
-
-RUN systemctl enable apache2.service
-
-EXPOSE 80
-```
-
-Build that image:
-
-```console
-$ docker built -t apache:test01 .
-```
-
-Create a footloose configuration file.
-
-```console
-$ footloose config create --image apache:test01
-```
-
-Now, create a machine!
-
-```console
-$ footloose create
-```
-
-Once the machine is ready, you should be able to access apache on the exposed port.
-
-```console
-$ docker port cluster-node0 80
-0.0.0.0:32824
-$ curl 0.0.0.0:32824
-<!DOCTYPE html>
-<html>
-    <title>Footloose</title>
-    <body>
-        Hello, from footloose!
-    </body>
-</html>
-```
-
-In case of multiple machines the port will be different on each machine.
-
-```console
-$ docker port cluster-node1 80
-0.0.0.0:32828
-
-$ docker port cluster-node0 80
-0.0.0.0:32826
-```
+To see some of the use cases, please refer to the Example's [README](./examples/README.md).
```

Comment:
It'd be nice to have a list of examples here with links to their README files:

- [Customize the OS image](examples/fedora29-htop/README.md)
- [Expose services](...)
- [Provision the VM with ansible](...)
- ....

Reply:
Yeah, I was wondering about which would look better.

Reply:
so you think the extra indirection is too much? Just link from here to the READMEs directly?